### ISSUE TITLE
Dovecot fails on Debian 10

### ISSUE THREAD

**Issue by henri-hulski** [hestiacp/hestiacp#644]
Similar problem as in #222 
On Debian 10 with Hestia recent dev build I get the following error in `dovecot.log`:
```
Error: namespace configuration error: Duplicate namespace prefix: ""
```
After removing `conf.d/15-mailboxes.conf` it worked.

**Comment by ghost** [comment]
Hi @henri-hulski,

Thanks for your report, we've confirmed this internally during our testing with Debian 10 and will be implementing a fix in the master branch before 1.1.0's release.

**Comment by ghost** [comment]
This should be fixed now in the installer for Debian 10.
